launcher/sdw_updater_gui/UpdaterAppUi.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+
+# Form implementation generated from reading ui file 'sdw_updater.ui'
+#
+# Created by: PyQt4 UI code generator 4.12.1
+#
+# WARNING! All changes made in this file will be lost!
+
+from PyQt4 import QtCore, QtGui
+
+try:
+    _fromUtf8 = QtCore.QString.fromUtf8
+except AttributeError:
+
+    def _fromUtf8(s):
+        return s
+
+
+try:
+    _encoding = QtGui.QApplication.UnicodeUTF8
+
+    def _translate(context, text, disambig):
+        return QtGui.QApplication.translate(context, text, disambig, _encoding)
+
+
+except AttributeError:
+
+    def _translate(context, text, disambig):
+        return QtGui.QApplication.translate(context, text, disambig)
+
+
+class Ui_UpdaterDialog(object):
+    def setupUi(self, UpdaterDialog):
+        UpdaterDialog.setObjectName(_fromUtf8("UpdaterDialog"))
+        UpdaterDialog.resize(520, 330)
+        self.layoutWidget = QtGui.QWidget(UpdaterDialog)
+        self.layoutWidget.setGeometry(QtCore.QRect(20, 20, 480, 300))
+        self.layoutWidget.setObjectName(_fromUtf8("layoutWidget"))
+        self.gridLayout = QtGui.QGridLayout(self.layoutWidget)
+        self.gridLayout.setMargin(0)
+        self.gridLayout.setObjectName(_fromUtf8("gridLayout"))
```

Comment:
nit: small thing but we should make it such that you cannot maximize the window (can be done in followup)